core/src/main/java/tech/tablesaw/columns/numbers/IntParser.java

```diff
@@ -7,6 +7,8 @@
 
 public class IntParser extends AbstractColumnParser<Integer> {
 
+  private boolean zeroDecimalAsFloat;
```

Comment:
this could be made `final` (same in the other parsers)

`zeroDecimalAsFloat` makes me think of the type `float`, which might be misleading. How about something like `ignoreZeroDecimal`?

Reply:
Agreed. Went back and forth there myself. Updated it now to `ignoreZeroDecimal`.

Reply:
Updated boolean variables to be `final` now.

In order to centralize the default value (`true`) and avoid maintaining it in several places, I added a constant to `ReadOptions`. Let me know if that is acceptable.

Reply:
This looks good to me.